src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -2451,11 +2454,9 @@ public Bundle tProblemConcernAct2Condition(ProblemConcernAct cdaProblemConcernAc
 				if(entry.getResource() instanceof Condition) {
 					Condition fhirCond = (Condition) entry.getResource();
 
-					// act/statusCode -> Condition.clinicalStatus
-					// NOTE: Problem status template is deprecated in C-CDA Release 2.1; hence status data is not retrieved from this template.
-					if(cdaProblemConcernAct.getStatusCode() != null && !cdaProblemConcernAct.getStatusCode().isSetNullFlavor()) {
-						fhirCond.setClinicalStatus(vst.tStatusCode2ConditionClinicalStatus(cdaProblemConcernAct.getStatusCode().getCode()));
```

Comment:
Can we back out tStatusCode2ConditionClinicalStatus function now?

Reply:
done